main/template/default/layout/footer.tpl

```diff
@@ -143,6 +143,25 @@
     });
     {% endraw %}
 
+    /**
+     * Generic function to replace the depreceted jQuery toogle function
```

Comment:
toggle, not toogle (makes it difficult to find all occurences of toggle later on)